config/application.rb

```diff
@@ -52,6 +52,9 @@ class Application < Rails::Application
 
     config.paths["log"] = ENV["LOG_PATH"] if ENV["LOG_PATH"]
 
+    # Google Analytics ID
+    config.universalId = ENV["GA_UNIVERSAL_ID"]
```

Comment:
I was gonna say this could be `ENV.fetch` to ensure it's always set (if for some reason the environment is defined), but what @kevindew suggested works too.

Reply:
you want to be careful of the ENV.fetch as they can cause you real pain points when you're building assets 😢